SentryCapacitor.podspec

```diff
@@ -17,15 +17,19 @@ Pod::Spec.new do |s|
 
   if File.exist?('../../@capacitor/core/package.json') == false
     # If Capacitor was not found (could happen when using Yarn PNP), fallback to the
-    # required minimum version of Capacitor 4.
-    miniOSVersion = '13.0'
+    # required minimum version of Capacitor 7.
+    miniOSVersion = '14.0'
   else
     capacitorPackage =  JSON.parse(File.read(File.join(__dir__, '../../@capacitor/core/package.json')))
     capacitorVersion = capacitorPackage['version']
-    if capacitorVersion.start_with?("2.") or capacitorVersion.start_with?("3.")
-      miniOSVersion = '12.0'
+    majorVersion = capacitorVersion.split('.').first.to_i
+
+    if majorVersion <= 3
+      miniOSVersion = '12.0' # Capacitor 3, 2.
+    elsif majorVersion < 7
+      miniOSVersion = '13.0' # Capacitor 6 or older.
     else
-      miniOSVersion = '13.0' # Required for Capacitor 4 and newer.
+      miniOSVersion = '14.0' # Capacitor 7 or higher.
```

Comment:
Does the Capacitor V7 fail to build without the matching minimum target?

I would have expected that a library could specify an older target than the framework.

Reply:
it did for me, the sdk was set with min iOS version 13 and it was complaining that the project required version 14.